ui/page/main_page.go

```diff
@@ -59,13 +60,18 @@ type MainPage struct {
 	usdExchangeSet  bool
 	totalBalance    dcrutil.Amount
 	totalBalanceUSD string
+
+	hideBalance *widget.Bool
+	text        string
+	hide        bool
```

Comment:
These struct variables are not needed. The `text` should be a local variable and you can get the current `hide` value from `mp.hideBalance.Value`.